_sass/_buttons.scss

```diff
@@ -13,24 +13,24 @@
   letter-spacing: 1px;
 }
 
-.button--primary {  
```

Comment:
oof, sorry man, haha, that's my code editor getting rid of whitespace it isn't into! There's quite a bit of this happening, I'm gonna try to remove all of 'em once we're wrapped!

Reply:
Actually I might leave them in. Most devs nowadays are using an editor that automatically patches this kind of thing, so it's likely that the next devs working on the site (me?? :smile: :smile:) will have the same kind of whitespace shenanigans on their own git commits.

I can explain more about this, and I can also really, really quickly (3 min flat, I bet you lunch) create a PR that patches all of the whitespace issues in the ebook.